### PR TITLE
Updating getting-started

### LINKED ISSUE #70: Update docs with LoadBalancer info
Show how to use deployment property for http source

### PR DESCRIPTION
**Pull request by trisberg**
- adding LoadBalancer example

Resolves #70

### CODE REVIEW

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -58,14 +58,12 @@ This is used by the locally running Data Flow Server to store the state of regis
...
+$ java -jar spring-cloud-dataflow-server-kubernetes-1.0.0.M2.jar --spring.cloud.deployer.kubernetes.memory=768Mi
 ```
 +
-If you are running Kubernetes using vagrant locally you might need to increase the CPU for the deployed apps using the following startup command:
+NOTE: We haven't tuned the memory use of the OOTB apps yet, so to be on the safe side we are increasing the memory for the pods by providing the following property: `--spring.cloud.deployer.kubernetes.memory=768Mi`
 +
-```
-$ java -jar spring-cloud-dataflow-server-kubernetes-1.0.0.M2.jar --spring.cloud.deployer.kubernetes.cpu=1
-```
+NOTE: If you are running Kubernetes using vagrant locally, then you might need to increase the CPU for the deployed apps using the following property: `--spring.cloud.deployer.kubernetes.cpu=1`
```
Comment (ilayaperumalg) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:66]: Is this really `1`? The default value for this property shows `250m` and that made me ask this question.

Comment (trisberg) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:66]: Default is 500m, where did you see 250m? Setting it to 1 is same as 1000m.

**Comment by trisberg** [comment]
Added a commit for server.adoc changes

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc (hunk 1 above)
Comment (ilayaperumalg) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:66]: yeah, that's right. I was wrong in mentioning `250m` as default.

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Comment by ilayaperumalg** [comment]
Squashed, rebased and merged as 14df388ab5fb9f06cb4fb9aa058cca526b5f414c along with the change to fix copyright and version property name in docs: 99b421209f5b50c6c6f346c8fd0cdbba4d4ce8f0

**Pull request closed without merging**